rails/config/environments/cucumber.rb

```diff
@@ -41,4 +41,9 @@
     # Disable logging to file. It might have performance impact while using Docker for Mac (slow filesystem sync).
     config.logger = Logger.new(STDOUT)
   end
+  # Specify developer chosen logging level. See docker-compose.yml and .env
+  # Using to something other than (0) debug makes test output more readable.
+  if (log_level = ENV['TEST_LOG_LEVEL'])
```

Comment:
Just realized this kind of comparison will probably fail if someone sets the ENV variable to an empty string. 
So it'd probably be better to change it to:
```
log_level = ENV['TEST_LOG_LEVEL']
if(log_level.present?)
  ...
```

Reply:
f6e7131 addresses this.